### ISSUE TITLE
Connect to jQuery draggable.

### ISSUE THREAD

**Issue by felipellrocha** [ducksboard/gridster.js#204]
It'd be nice to be able to connect gridster to jQuery draggable. That way, the developer could create a list of possible widgets the user could drag into gridster, and from then on, gridster would work as normal.

**Comment by devthejo** [comment]
I agree, it's blocking me to use it for a project I'm working on, that would be a nice feature!
